packages/editor/src/components/rich-text/README.md

```diff
@@ -57,6 +57,25 @@ Render a rich [`contenteditable` input](https://developer.mozilla.org/en-US/docs
 
 *Optional.* A list of autocompleters to use instead of the default.
 
+### `annotations: Array`
+
+*Optional.* A list of annotations that should be applied to the content.
+
+An annotation has the following shape:
+
+```js
+{
+	// The source of the annotation, will be used for the HTML class.
+	"source": "annotations-tester",
```

Comment:
How does the plugin identify the annotation if there's only a the source plugin, no ID?

Reply:
The idea with the source is that one can easily remove all annotations by a certain source. The ID is if plugins want to track specific annotations and do work based on that.